### PR TITLE
Update realm-export.json

### PR DESCRIPTION
**Pull request by Valecant**
new realm config

## Proposed changes

Describe the big picture of your changes here to communicate to the
maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to
that issue.

## Types of changes

What types of changes does your code introduce to the project: _Put
an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
      functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after
creating the PR. If you're unsure about any of them, don't hesitate to
ask. We're here to help! This is simply a reminder of what we are going
to look for before merging your code._

- [ ] I have read the [CONTRIBUTING][contrib] doc
- [ ] I have signed the [Contributor License Agreement][cla]
- [ ] I have updated the [RELEASE NOTES][release]
- [ ] I have added tests that prove my fix is effective or that my
      feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in
      downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion
by explaining why you chose the solution you did and what alternatives
you considered, etc...

[cla]: https://raw.githubusercontent.com/orchestracities/anubis/master/individual_cla.pdf
    "Martel Open Source Software Individual Contributor License Agreement"
[contrib]: https://github.com/orchestracities/anubis/blob/master/CONTRIBUTING.md
    "Contributing to Anubis"
[release]: https://github.com/orchestracities/anubis/blob/master/RELEASE_NOTES.md
    "Anubis Release Notes"

### CODE REVIEW

**Comment by chicco785** [comment]
@Valecant as by error checks and tue discussion, can you:
- sign the PR https://github.com/orchestracities/anubis/pull/46#issuecomment-1066898087
- update the RELEASE_NOTES.md

**Comment by Valecant** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Valecant** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by chicco785** [comment]
@Valecant 

1) there was a pr open with the cors support, don't pollute PRs
2) after you update the PR, check it passes, if not fix what's wrong.

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Anubis Release Notes
 
+### Change the main py in order to work with the frontend app
```
Comment (chicco785) [inline RELEASE_NOTES.md:3]: follow the existing structure, don't mess up.
1 - cors is not in this pr, so you can remove it.
2 - simply add the change description to real-export (what was changed), under new features in this case for 0.0.3

File: anubis-management-api/src/main.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from fastapi import Depends, FastAPI
 from tenants import routers as t
 from policies import routers as p
+from fastapi.middleware.cors import CORSMiddleware
```
Comment (chicco785) [inline anubis-management-api/src/main.py:4]: as said above, following gabriele's pr, code in the master has already cors support.
so remove.

**Review by chicco785 — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -1,5 +1,20 @@
 # Anubis Release Notes
 
+## 0.0.4
```
Comment (chicco785) [inline RELEASE_NOTES.md:3]: non creare una nuova versione, fin tanto che non facciamo una release, rimane 0.0.3

**Review by chicco785 — CHANGES_REQUESTED** [review]
1. use 0.0.3, add rows there, not a new version
2. remove `.DS_STORE` files

**Review by chicco785 — APPROVED** [review]

**Pull request merged**